gcp_iam_roles/dynatrace-gcp-function-deployment-role.yaml

```diff
@@ -0,0 +1,44 @@
+# create with command:
+# gcloud iam roles create dynatrace_function.deployment --project={PROJECT_ID} --file=dynatrace-gcp-function-deployment-role.yaml
+title: Dynatrace GCP Function deployment role
+description: Role for Dynatrace GCP function helm and pubsub deployment
+stage: GA
+includedPermissions:
+  - pubsub.subscriptions.get
```

Comment:
from pubsub set of permissions seem like only   pubsub.subscriptions.get (maybe also pubsub.subscriptions.list) is needed for deploy-helm script.
Other pubsub permissions should be moved into e2e-tests role

Reply:
no, all of this are required. Our documentation has two files `deploy-helm.sh` and `deploy-pubsub.sh`

Reply:
that other settings are required by that second script. It was tested